docs/checkers/index.md

```diff
@@ -482,6 +482,16 @@ This error occurs when a non-mapping value is used in a place where mapping is e
 
 ```
 
+(W1117)=
+
+### Keyword argument superseded by positional argument (W1117)
+
+This error occurs when a function is called with keyword arguments with the same name as positional-only parameters and the function contains a keyword variadic parameter dict.
```

Comment:
It would be good to refer to the example you created specifically, to explain why this is a problem (similar to what you did in yesterday's presentation). E.g., "In the example below, ..."

Then, show how you would call the function to pass in `'Hi'` as the greeting (without making it a keyword argument)